sark/code/function.py

```diff
@@ -50,6 +80,81 @@ def __repr__(self):
             repeat=repr(self.repeat))
 
 
+class FunctionFlags(long):
```

Comment:
I don't like inheriting from `long`.
That said, I'm not sure how to handle the current situation without breaking old code. Maybe a new name for this member?

Reply:
I think a new name is the only other way to maintain backwards compatibility. I can't think of a good name for it, though. The only thing I can think of is `flag_checks` and that seems overly verbose.  Any suggestions?

Reply:
Also, If you don't mind me asking, why don't you like inheriting from `long`?  I'm not trying to disagree with you, just trying to learn.